tests/test_destroy.js

```diff
@@ -1,25 +1,27 @@
 'use strict'
 
-const nock = require('..')
+const { expect } = require('chai')
 const http = require('http')
 const { test } = require('tap')
+const nock = require('..')
 
 require('./cleanup_after_each')()
+require('./setup')
 
 test('req.destroy should emit error event if called with error', t => {
   nock('http://example.test')
     .get('/')
     .reply(404)
 
+  const respErr = new Error('Response error')
+
   http
     .get('http://example.test/', res => {
-      if (res.statusCode !== 200) {
-        res.destroy(new Error('Response error'))
-      }
+      expect(res.statusCode).to.equal(404)
+      res.destroy(respErr)
     })
     .once('error', err => {
-      t.type(err, Error)
-      t.equal(err.message, 'Response error')
+      expect(err).to.equal(respErr)
```

Comment:
Is there an assertion that does the same as `Object.is`?

Reply:
There is not unfortunately. 

`equal` uses strict equality under the hood, so since the variable isn't a scalar it should have the same affect as `object.is`

To confirm, I tried 
```js
const respErr = new Error('Response error')
const respErr2 = new Error('Response error')

...

expect(err).to.equal(respErr)
expect(err).to.equal(respErr2)
```
And this failed on the second assertion.

I could aways add `expect(obeject.is(err, respErr).to.be.true()` if you think it gets the point across better.

Reply:
It might be a nice thing to add to Chai, though I think it's okay to use `.equal()`.

Reply:
It was brought up a few years ago, but closed due to lack of interest. https://github.com/chaijs/chai/issues/708